lib/dkim.js

```diff
@@ -463,8 +463,9 @@ function DKIMVerify(email, callback, sigs) {
 
         // The other DNS errors are operational in nature;
         sigs.result = sig.result = false;
-        sigs.issue_desc = sig.issue_desc = err;
-        return callback(err, sigs);
+        sigs.issue_desc = sig.issue_desc = err.message;
+
+        return callback(null, sigs); // returning the error AND the sigs breaks promisification... should do callback(null, sigs)
```

Comment:
Looks like the nodejs DNS library has some good docs on this, since `dns.resolveTxt` is a sugar alias for `dns.resolve` ---> https://nodejs.org/api/dns.html#dns_dns_resolve_hostname_rrtype_callback

which says:
>On error, err is an Error object, where err.code is one of the error codes listed [here](https://nodejs.org/api/dns.html#dns_error_codes).

So maybe just checking for existence of `err.code`?